src/main/java/cloud/fogbow/fs/constants/ApiDocumentation.java

```diff
@@ -0,0 +1,102 @@
+                " operation and a description of the operation.";
+	}
+	
+	public static class Plan {
+	    public static final String API = "Manages operations on finance plans";
+        public static final String UPDATE_FINANCE_STATE_OPERATION = "Updates the given user's finance state using given state map.";
+        public static final String UPDATE_FINANCE_STATE_REQUEST_BODY = "The body of the request must contain the finance state as a map.";
+        public static final String GET_FINANCE_STATE_PROPERTY_OPERATION = "Returns a property of the given user's finance state.";
+        public static final String GET_FINANCE_STATE_PROPERTY_REQUEST_BODY = "The body of the request must contain a public key, used to encrypt the result.";
+        public static final String CREATE_FINANCE_PLAN_OPERATION = "Creates a finance plan.";
+        public static final String CREATE_FINANCE_PLAN_REQUEST_BODY = "The body of the request must contain the parameters used to create the finance plan:" +
+                " the class name of the finance plugin used to instantiate the finance plan, the finance plan name and the other finance plan options as a map.";
+        public static final String GET_FINANCE_PLAN_OPERATION = "Returns a finance plan's properties.";
+        public static final String CHANGE_FINANCE_PLAN_OPTIONS_OPERATION = "Updates the options used by a finance plan.";
+        public static final String CHANGE_FINANCE_PLAN_OPTIONS_REQUEST_BODY = "The body of the request must contain the finance plan options as a map.";
+        public static final String REMOVE_FINANCE_PLAN_OPERATION = "Removes a finance plan.";
+        public static final String PROPERTY = "A string describing the user's finance state property.";
+        public static final String USER_ID = "The ID of the specific user.";
+        public static final String PROVIDER = "The ID of the specific identity provider.";
+        public static final String PLAN_NAME = "The name of the specific plan.";
+	}
+	
+	public static class User {
+	    public static final String API = "Manages operations that can be performed by any user.";
+        public static final String REGISTER_SELF_OPERATION = "Registers the user represented by the user token to be financially managed by the given finance plan.";
+        public static final String UNREGISTER_SELF_OPERATION = "Unregisters the user represented by the user token from the finance plan it currently uses." + 
+                " The user financial state will remain unmanaged until the user is registered again to a finance plan.";
+        public static final String CHANGE_SELF_PLAN_OPERATION = "Changes the plan used to manage the user represented by the user token.";
+        public static final String PLAN_NAME = "The name of the specific plan.";
+	}
+
+	// TODO the string examples for PLAN_INFO and OPERATION need improvements
```

Comment:
Fix the ToDos

Reply:
The issue here is Swagger does not process the example string correctly. It must be presented as a regular string containing a map but it shows only as a map and I don't know how to fix this.